packages/credential-data-integrity/README.md

```diff
@@ -0,0 +1,101 @@
+# Data Integrity Proof Plugin
+
+This package draws inspiration from [eddsa-rdfc-2022-cryptosuite](@digitalbazaar/eddsa-rdfc-2022-cryptosuite) and is implemented as a plugin for [Veramo](https://veramo.io/). It adheres to the [Data Integrity EdDSA Cryptosuites v1.0](https://www.w3.org/TR/vc-di-eddsa/) specification to generate a Data Integrity proof for credentials.
+
+This package contains a plugin and a message handler for issuing and verifying credentials that adhere to W3C standards.
+
+## Setup
+
+- Declare the plugin in your Veramo agent:
+
+```yaml
+credentialIssuerLD:
+  $require: '@veramo/credential-ld#CredentialIssuerLD'
+  $args:
+    - suites:
+        # - $require: '@veramo/credential-ld#VeramoEd25519Signature2018'
+        # - $require: '@veramo/credential-ld#VeramoEd25519Signature2020'
+        # - $require: '@veramo/credential-ld#VeramoJsonWebSignature2020'
+        # - $require: '@veramo/credential-ld#VeramoEcdsaSecp256k1RecoverySignature2020'
+        - $require: '@vckit/credential-data-integrity#VCkitEddsaRdfc2022'
+      contextMaps:
+        - $require: '@vckit/credential-data-integrity?t=object#contexts'
+        - $require: '@veramo/credential-ld?t=object#LdDefaultContexts'
+        - $require: '@transmute/credentials-context?t=object#contexts'
+        - $require: '@transmute/did-context?t=object#contexts'
+        - $require: '@vckit/renderer?t=object#RenderDefaultContexts'
+        - $require: '@vckit/revocationlist?t=object#RevocationListDefaultContexts'
+        #  others should be included here
+```
+
+## Issue the Credential
+
+You can issue a new verifiable credential by using the `/agent/routeCreationVerifiableCredential` API endpoint with the proof format set to `lds`. The issuer can be a `did:web`.
+
+The example below shows how to issue a credential with the Data Integrity proof plugin. You can replace the `issuer` and `credentialSubject` with your own data.
+
+```curl
+curl --request POST \
+  --url http://localhost:3332/agent/routeCreationVerifiableCredential \
+  --header 'Content-Type: application/json' \
+  --data '{
+	"credential": {
+		"@context": [
+			"https://www.w3.org/2018/credentials/v1",
```

Comment:
Can it use the V2 VCDM context?

Reply:
Yes, it can be used with the V2 VCDM context, I have updated this README to reflect that.